### PR TITLE
Fix can't scroll email content in browser

### PR DESCRIPTION
**Pull request by dab246**
https://user-images.githubusercontent.com/80730648/158779033-0f951365-ec52-43b1-8d3c-5c8d1ce940a7.mov

### CODE REVIEW

File: analysis_options.yaml [hunk 1]
```diff
@@ -7,8 +7,6 @@
 
 # The following line activates a set of recommended lints for Flutter apps,
 # packages, and plugins designed to encourage good coding practices.
-include: package:flutter_lints/flutter.yaml
```
Comment (hoangdat) [inline analysis_options.yaml:10]: why you remove that?

Comment (dab246) [reply analysis_options.yaml:10]: It's not necessary here

Comment (hoangdat) [reply analysis_options.yaml:10]: We need to have lint check, IMO we need it :)

Comment (dab246) [reply analysis_options.yaml:10]: The problem here is that we have not added library lint so it is always not found when doing `flutter analyze`

Comment (hoangdat) [reply analysis_options.yaml:10]: okie, move forward, we will back on that problem later : #367

**Review by hoangdat — APPROVED** [review]

**Pull request merged**